logstreams/src/test/java/io/zeebe/logstreams/state/FailingSnapshotChunkReplicationTest.java

```diff
@@ -105,48 +103,48 @@ public void shouldNotMarkSnapshotAsValidIfNotReceivedAllChunks() throws Exceptio
     final List<SnapshotChunk> replicatedChunks = replicator.replicatedChunks;
     assertThat(replicatedChunks.size()).isGreaterThan(0);
 
-    final File snapshotDirectory = receiverStorage.getTmpSnapshotDirectoryFor("1");
+    final var snapshotDirectory = receiverStorage.getPendingDirectoryFor("1");
     assertThat(snapshotDirectory).exists();
-    final File[] files = snapshotDirectory.listFiles();
+    final var files = Files.list(snapshotDirectory).collect(Collectors.toList());
     assertThat(files)
-        .extracting(File::getName)
+        .extracting(p -> p.getFileName().toString())
         .containsExactlyInAnyOrder(
             replicatedChunks.subList(0, 2).stream()
                 .map(SnapshotChunk::getChunkName)
                 .toArray(String[]::new));
-    assertThat(receiverStorage.existSnapshot(1)).isFalse();
+    assertThat(receiverStorage.exists("1")).isFalse();
   }
 
-  @Test
-  public void shouldDeleteOrphanedSnapshots() throws Exception {
-    // given
-    final FlakyReplicator flakyReplicator = new FlakyReplicator();
-    setup(flakyReplicator);
-    receiverSnapshotController.consumeReplicatedSnapshots();
-    replicatorSnapshotController.takeSnapshot(1);
-    replicatorSnapshotController.replicateLatestSnapshot(Runnable::run);
-    replicatorSnapshotController.close();
-
-    final Replicator workingReplicator = new Replicator();
-    setupReplication(workingReplicator);
-    receiverSnapshotController.consumeReplicatedSnapshots();
-    replicatorSnapshotController.takeSnapshot(2);
-    replicatorSnapshotController.replicateLatestSnapshot(Runnable::run);
-
-    // when
-    replicatorSnapshotController.takeSnapshot(3);
-    replicatorSnapshotController.replicateLatestSnapshot(Runnable::run);
-
-    // then
-    final List<SnapshotChunk> replicatedChunks = workingReplicator.replicatedChunks;
-    assertThat(replicatedChunks.size()).isGreaterThan(0);
-
-    final File snapshotDirectory = receiverStorage.getTmpSnapshotDirectoryFor("1");
-    assertThat(snapshotDirectory).doesNotExist();
-    assertThat(receiverStorage.existSnapshot(1)).isFalse();
-    assertThat(receiverStorage.existSnapshot(2)).isFalse();
-    assertThat(receiverStorage.existSnapshot(3)).isTrue();
-  }
+  //  @Test
+  //  public void shouldDeleteOrphanedSnapshots() throws Exception {
+  //    // given
+  //    final FlakyReplicator flakyReplicator = new FlakyReplicator();
+  //    setup(flakyReplicator);
+  //    receiverSnapshotController.consumeReplicatedSnapshots();
+  //    replicatorSnapshotController.takeSnapshot(1);
+  //    replicatorSnapshotController.replicateLatestSnapshot(Runnable::run);
```

Comment:
This need to be fixed I think

Reply:
Removed since this was moved to the broker with the snapshot store/storage impl